python/ray/tune/trial.py

```diff
@@ -7,6 +7,7 @@
 import time
 import tempfile
 import os
+from dict_hash import sha256
```

Comment:
any other way to do this?

Reply:
I can move the code that I use in that package, but that does not feel like very pythonic. I am not using directly the python hash since it is not consistent between instances and reboots, and needs to be invariant to what regards the keys ordering.